data/pages-content/photos.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Photos
+subtitle: Savour, relive, share the highlights of EuroPython 2022
+---
+
+# EuroPython 2022 Photos
+
+### Official Photos
+<ButtonWithTitle title="taken by Larissa Oliveira & Liam McCafferty" text="EuroPython 2022 Official Photos!" href="https://www.flickr.com/groups/14792291@N24/" />
```

Comment:
Can we please add a social call back to both of them so that people can contact them for future opportunities? :)

Reply:
Added in https://github.com/EuroPython/website/pull/206/commits/0125ae4b59019fed983d71064e048a6722e3eee7